### ISSUE TITLE
Don't set "<"...">" as bracket pair

### ISSUE THREAD

**Issue by hediet** [michaelgmcd/vscode-language-babel#111]
**Describe the issue**

This is the current language configuration that babel ships for jsx:
```json
{
  "comments": {
    "blockComment": ["{/*", "*/}"]
  },
  "brackets": [["<", ">"], ["{", "}"], ["[", "]"], ["(", ")"]],
  "colorizedBracketPairs": [["{", "}"], ["[", "]"], ["(", ")"]],
  "autoClosingPairs": [
    { "open": "{", "close": "}" },
    { "open": "[", "close": "]" },
    { "open": "(", "close": ")" },
    { "open": "'", "close": "'", "notIn": ["string", "comment"] },
    { "open": "\"", "close": "\"", "notIn": ["string"] },
    { "open": "/**", "close": " */", "notIn": ["string"] }
  ],
  "surroundingPairs": [
    ["{", "}"],
    ["[", "]"],
    ["(", ")"],
    ["<", ">"],
    ["'", "'"],
    ["\"", "\""]
  ]
}
```

Because `["<", ">"]` is configured as bracket pair, VS Code tries to match these brackets, but only if they have the same language id.

Unfortunately, this causes red brackets since the latest update (because we consider "brackets" again, even if "colorizedBracketPairs" is set):

![image](https://user-images.githubusercontent.com/2931520/222697297-5b4e7ce8-8543-4dbc-ad83-4c0c980f7907.png)

This is because the language id of the opening bracket and closing bracket don't agree:

![image](https://user-images.githubusercontent.com/2931520/222697353-92ec7d7d-e021-4b54-858e-14c780fda4dd.png)

![image](https://user-images.githubusercontent.com/2931520/222697430-97333b05-8148-42dd-8cc7-a2eb0816e041.png)

**Sample Code to Reproduce**
```jsx
import React from "react";
import Display from "./Display";
import ButtonPanel from "./ButtonPanel";
import calculate from "../logic/calculate";
import "./App.css";

export default class App extends React.Component {
  state = {
    total: null,
    next: null,
    operation: null,
  };

  handleClick = buttonName => {
    this.setState(calculate(this.state, buttonName));
  };

  render() {
    return (
      <div className="component-app">
        <Display value={this.state.next || this.state.total || "0"} />
        <ButtonPanel clickHandler={this.handleClick} />
      </div>
    );
  }
}
```

**Possible Solution**

I recommend to just remove the "<" ... ">" bracket pair.

**Comment by dpidan** [comment]
> Unfortunately, this causes red brackets since the latest update (because we consider "brackets" again, even if "colorizedBracketPairs" is set):

Small clarification for readers that this is from the latest [VS Code [1.76.0] update](https://code.visualstudio.com/updates/v1_76#_editor)

Until this is addressed, a workaround is possible by setting brackets in your VS Code user `settings.json`:

```json
{
    "[jsx]": {
        "editor.language.brackets": [
            ["{", "}"], ["[", "]"], ["(", ")"]
        ]
    },
    "[jsx-attr]": {
        "editor.language.brackets": [
            ["{", "}"], ["[", "]"], ["(", ")"]
        ]
    }
}
```

**Comment by michaelgmcd** [comment]
Thanks for the detailed issue, taking a look at this now.

**Comment by michaelgmcd** [comment]
Should be fixed in v0.0.37
